src/OrchardCore.Themes/TheComingSoonTheme/wwwroot/gulpfile.js

```diff
@@ -10,7 +10,7 @@ const header = require("gulp-header");
 const merge = require("merge-stream");
 const plumber = require("gulp-plumber");
 const rename = require("gulp-rename");
-const sass = require("gulp-sass");
+const sass = require("gulp-dart-sass");
```

Comment:
These might get overwritten when @agriffard will update the theme.

Reply:
Yeah was thinking might just pr the source to upgrade it as well. To save that issue.